fastlane/Helpers/Distribution.swift

```diff
@@ -25,4 +25,27 @@ enum Distribution {
             debug: .userDefined(true)
         )
     }
+
+    static func uploadToAppStore() {
+        let environment: Constant.Environment = .production
+        let ipaPath = makeIpaPath(environment: environment)
+        appStoreConnectApiKey(
```

Comment:
Let's separate this into a new function. I might use this for downloading the dSYM file. How about create

```swift
enum AppStoreAuthentication { static func connectAPIKey() }
```